pkg/api/validation/validation.go

```diff
@@ -1481,7 +1483,15 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 			allErrs = append(allErrs, field.Invalid(specPath.Child("storageClassName"), pv.Spec.StorageClassName, msg))
 		}
 	}
-
+	if !utilfeature.DefaultFeatureGate.Enabled(features.BlockVolumeSupport) {
```

Comment:
You should only check for the feature gate if VolumeMode is set.  Otherwise, you are requiring the feature gate to be set even if you don't use it.